desktop/src/renderer/index.html

```diff
@@ -10,7 +10,7 @@
     />
   </head>
 
-  <body>
+  <body style="min-height:100vh;min-width:100vh">
```

Comment:
I think we should just keep the styling for body in the css. However the body styles in main.css can be combined with the body styles in base.css.